ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt

```diff
@@ -0,0 +1,158 @@
+# The Tower Of Babil
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839::.*is no longer sealed/ window 100000 jump 0
+
+
+#~~~~~~~~~~#
+# BARNABAS #
+#~~~~~~~~~~#
+
+# -ii 6076 61DE
+
+0.0 "--sync--" sync / 00:0839::Magitek Servicing will be sealed off/ window 0,1
+8.9 "Ground and Pound" sync / 1[56]:[^:]*:Barnabas:6247:/ window 8.9,5
+27.3 "Ground and Pound" sync / 1[56]:[^:]*:Barnabas:62EA:/
+
+40.5 "Dynamic Scrapline" sync / 1[56]:[^:]*:Barnabas:(6246|62F0):/ window 40.5,10
+41.6 "--sync--" sync / 1[56]:[^:]*:Barnabas:61DD:/
+43.0 "Electromagnetic Release" sync / 1[56]:[^:]*:Barnabas:62F1:/
+55.2 "Shocking Force" sync / 1[56]:[^:]*:Barnabas:62EC:/
+72.2 "Dynamic Pound" sync / 1[56]:[^:]*:Barnabas:(6245|62EE):/ window 72.2,10
+73.3 "--sync--" sync / 1[56]:[^:]*:Barnabas:6075:/
+74.7 "Electromagnetic Release" sync / 1[56]:[^:]*:Barnabas:62EF:/
+85.3 "Thundercall" sync / 1[56]:[^:]*:Barnabas:62ED:/ window 85.3,10
+91.8 "Rolling Scrapline" sync / 1[56]:[^:]*:Barnabas:62EB:/
```

Comment:
Does Rolling Scrapline have two ids as well?

Reply:
Not that I saw across both our logs, and there's only one entry on [XIVAPI](https://xivapi.com/search?indexes=action&string=rolling+scrapline&columns=Name,Name_de,Name_fr,Name_ja,ID).